dot/network/transaction.go

```diff
@@ -119,36 +120,54 @@ func decodeTransactionHandshake(_ []byte) (Handshake, error) {
 	return &transactionHandshake{}, nil
 }
 
-func (s *Service) createBatchMessageHandler(txnBatch chan *BatchMessage) NotificationsMessageBatchHandler {
-	return func(peer peer.ID, msg NotificationsMessage) (msgs []*BatchMessage, err error) {
+func (s *Service) createBatchMessageHandler(txnBatchCh chan *BatchMessage) NotificationsMessageBatchHandler {
+	go func() {
+		protocolID := s.host.protocolID + transactionsID
+		ticker := time.NewTicker(s.cfg.SlotDuration)
+		defer ticker.Stop()
+
+		for {
+		out:
```

Comment:
Please get rid of the break + label, it adds unneeded complexity and is rather uncommon in Go code.

You can just use boolean and some `continue`/ `break` to control the flow.

Reply:
Done

Reply:
I don't think it's up yet, did you push the changes?